go/membufc/fields_extended.go

```diff
@@ -0,0 +1,26 @@
+// Copyright 2018 the membuffers authors
+// This file is part of the membuffers library in the Orbs project.
+//
+// This source code is licensed under the MIT license found in the LICENSE file in the root directory of this source tree.
+// The above notice should be included in all copies or substantial portions of the software.
+
+package main
+
+import "github.com/orbs-network/pbparser"
+
+func isExtendedType(name string) bool {
+	return name == "membuffers.bytes32"
+}
+
+func getExtendedType(messageName string, field pbparser.FieldElement, isArray bool) MessageField {
```

Comment:
you should probably check that the field type is indeed `bytyes32` and otherwise return an error